test/system/src/main/java/io/pravega/test/system/framework/services/marathon/BookkeeperService.java

```diff
@@ -96,7 +96,7 @@ private App createBookieApp() {
         app.getContainer().getDocker().setImage(IMAGE_PATH + "/nautilus/bookkeeper:" + PRAVEGA_VERSION);
         Collection<Volume> volumeCollection = new ArrayList<>();
         volumeCollection.add(createVolume("/bk", "mnt", "RW"));
-        //TODO: add persistent volume  (see issue https://github.com/pravega/pravega/issues/639)
+        volumeCollection.add(createPersistentVolume("persistent", "RW"));
```

Comment:
The container path seems incorrect. We would need to pass $`USE_MOUNT` to change `BK_DIR `from /bk. (ref: `entrypoint.sh` for Bookkeeper docker image).
Also, we need to finalize  a. if a local persistent volume will be used b. its size. 
ref: https://mesosphere.github.io/marathon/docs/persistent-volumes.html

Reply:
Will update the PR, once we decide upon if a persistent volume is needed.